packages/embeds/embed-core/src/preview.ts

```diff
@@ -2,18 +2,17 @@ const WEBAPP_URL =
   import.meta.env.EMBED_PUBLIC_WEBAPP_URL || `https://${import.meta.env.EMBED_PUBLIC_VERCEL_URL}`;
 const EMBED_LIB_URL = import.meta.env.EMBED_PUBLIC_EMBED_LIB_URL || `${WEBAPP_URL}/embed/embed.js`;
 
-(window as any).fingerprint = import.meta.env.EMBED_PUBLIC_EMBED_FINGER_PRINT as string;
-
 // Install Cal Embed Code Snippet
 (function (C, A, L) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This is the snippet included in preview.ts. It is already typed well in embed-snippet, so no need to focus on types here.